scripts/station_csv_to_json.py

```diff
@@ -18,6 +29,10 @@
 
     WEATHER_STATIONS = []
 
+    ECODIVISIONS = geopandas.read_file('app/data/ERC_ECODIV_polygon/ERC_ECODIV_polygon.shp')
+    with open('app/data/ecodivisions_core_seasons.json') as file_handle:
```

Comment:
Can we keep this out of the app directory also? I don't see it referenced anywhere that the API needs it right now. (Just to keep what we deploy to the server to a minimum)